tests/providers/amazon/aws/operators/test_sagemaker_transform.py

```diff
@@ -50,15 +50,16 @@
     "ExecutionRoleArn": "arn:aws:iam:role/test-role",
 }
 
-CONFIG: dict = {"Model": CREATE_MODEL_PARAMS, "Transform": CREATE_TRANSFORM_PARAMS}
 
-
-class TestSageMakerTransformOperator(unittest.TestCase):
-    def setUp(self):
+class TestSageMakerTransformOperator:
+    def setup_method(self):
+        self.create_transform_params = copy.deepcopy(CREATE_TRANSFORM_PARAMS)
+        self.create_model_params = copy.deepcopy(CREATE_MODEL_PARAMS)
+        self.config = {"Model": self.create_model_params, "Transform": self.create_transform_params}
```

Comment:
After migrate to `pytest` some test failed due to mutability of test parameters so this parameters recreate for each test case.